Community/rest_api/dns_page.py

```diff
@@ -524,11 +526,13 @@ def patch(self, configuration, view, absolute_name):
             return 'No matching Host Record(s) found', 404
         if data['properties'] is not None:
             properties = data.get('properties')
-            host_record.properties = util.properties_to_map(properties)
+            host_record._properties = util.properties_to_map(properties)
```

Comment:
is there no proper way to update this without updating a _ attr?

Reply:
Because in entity.py have no function properties so we can't use host_record with attr properties.